src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  padding: 0 50px;
+}
+
+.logo__image {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin: 0;
+}
+
+.nav__link {
+  display: block;
+  position: relative;
+  font-family: Roboto, sans-serif;
+  color: #000;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav__link:link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  cursor: pointer;
+}
+
+.is-active,
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active:after {
```

Comment:
to create `after` pseudoelement we need to use `::` instead of `:` which is used for pseudoclasses